### PR TITLE
Flp 198 get apprentice names api

### PR DESCRIPTION
**Pull request by dinathain**
Added new client, service etc. for for fetching all apprenticeships from the Apprenticeship inner API (from the Funding Outer API).

### CODE REVIEW

File: src/Funding/SFA.DAS.Funding.Api/Models/GetApprenticeshipsResponse.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            {
+                apprenticeships.Add(new ApprenticeshipDto()
+                {
+                    Uln = apprenticeship.Uln,
+                    FirstName = apprenticeship.FirstName,
+                    LastName = apprenticeship.LastName
+                });
+            }
+
+            Apprenticeships = apprenticeships;
```
Comment (tom-gough) [inline src/Funding/SFA.DAS.Funding.Api/Models/GetApprenticeshipsResponse.cs:23]: I maybe would have done this, up to you if you prefer it:

`Apprenticeships = source.Select(x => new ApprenticeshipDto
            {
                FirstName = x.FirstName,
                LastName = x.LastName,
                Uln = x.Uln
            });`

File: src/Funding/SFA.DAS.Funding.UnitTests/InnerApi/Requests/Apprenticeships/WhenBuildingTheGetApprenticeshipsRequest.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.Funding.InnerApi.Requests.Apprenticeships;
+
+namespace SFA.DAS.Funding.UnitTests.InnerApi.Requests.ProviderEarnings
```
Comment (tom-gough) [inline src/Funding/SFA.DAS.Funding.UnitTests/InnerApi/Requests/Apprenticeships/WhenBuildingTheGetApprenticeshipsRequest.cs:6]: Namespace needs correcting

**Review by tom-gough — CHANGES_REQUESTED** [review]
Namespace needs correcting, see comments. Otherwise looks good.

File: src/Funding/SFA.DAS.Funding.Api/Models/GetApprenticeshipsResponse.cs (hunk 1 above)
Comment (dinathain) [reply src/Funding/SFA.DAS.Funding.Api/Models/GetApprenticeshipsResponse.cs]: Good point - fixed.

File: src/Funding/SFA.DAS.Funding.UnitTests/InnerApi/Requests/Apprenticeships/WhenBuildingTheGetApprenticeshipsRequest.cs (hunk 2 above)
Comment (dinathain) [reply src/Funding/SFA.DAS.Funding.UnitTests/InnerApi/Requests/Apprenticeships/WhenBuildingTheGetApprenticeshipsRequest.cs]: Fixed

**Review by tom-gough — APPROVED** [review]

**Pull request merged**